components/arrow-button.scss

```diff
@@ -0,0 +1,32 @@
+.ArrowButton{
+  cursor: pointer;
+  width: 40px;
+  height:48px;
+  background: linear-gradient(136deg, #001358, #125ac4);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  opacity: 0.3;
+  transition: all 0.25s;
+
+  &:hover{
+    opacity: 1;
+    transition: all 0.25s;
```

Comment:
Cool, anything over `180ms` is good since is the minimum timeframe the human eye needs in order to perceive motion.